src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/CollectBaseDistributionByCycleSparkIntegrationTest.java

```diff
@@ -78,6 +78,50 @@ public void test(final String unsortedBamName, final String referenceName, final
         IntegrationTestSpec.assertEqualTextFiles(outfile, expectedFile, "#");
     }
 
+    @Test
+    public void testInGatkLiteDocker() throws IOException {
+        final String gatkLiteDockerProperty = System.getProperty("IN_GATKLITE_DOCKER");
+
+        try {
+            System.setProperty("IN_GATKLITE_DOCKER", "true");
+
+            final File unsortedBam = new File(TEST_DATA_DIR, "unsortedBamName");
+
+            final File outfile = GATKBaseTest.createTempFile("test", ".metrics");
+            final File pdf = GATKBaseTest.createTempFile("test", ".pdf");
+
+            ArgumentsBuilder args = new ArgumentsBuilder();
+            args.addRaw("-" + "I");
```

Comment:
This doesn't need to change but in the future (lol...) I would use the specialty fluent methods on args.

```
          ArgumentsBuilder args =  ArgumentsBuilder.create()
                    .addInput(unsortedBam)
                    .addOutput(outfile)
                    .add("chart", pdf)
                    .add("pf-reads-only", false)
                    .add("aligned-reads-only", false);
```